addons/website_slides_sale/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_website_slides_sale_website_slides_sale,website_slides_sale.website_slides_sale,model_website_slides_sale_website_slides_sale,,1,0,0,0
```

Comment:
What is this access right ?

Reply:
I should be more careful with 'scaffold' apparently, I removed it.